php/user's-teams.php

```diff
@@ -0,0 +1,47 @@
+<?php
+///////////////////////////////////////////////////
+/// Return an array of objects with the following
+/// structure:
+/// id: team's id
+/// name: team's name
+/// members: number of team's member
+///////////////////////////////////////////////////
+	require 'connect-db.inc';
+	session_start();
+
+	$db = connect_db();
+	$user_id = $_SESSION ['user_id'];
+
+	$stmt = $db->prepare('SELECT collaborate.team_id, team.name FROM collaborate JOIN team
```

Comment:
You didn't any change here. You don't need to use JOIN here, just get name and id from team table